### ISSUE TITLE
The keras.layere.reshape() is a dynamic-sized tensor ? convert CRNN model to TFLite cannot work on android GPU

### ISSUE THREAD

**Issue by Yunlei-AI** [tensorflow/tensorflow#52616]
### 1. System information

- Linux Ubuntu 20.04,
- android API 25
- tf-nightly   keras=2.4.3

### 2. Code

CRNN model use rashape like this:

     x = layers.Reshape((4, 128))(x)

Convert code:

    keras_model = models.load_model(model_path, custom_objects={ 'precision': precision, 'recall': recall, 'f_score': f_score})
    converter = tf.lite.TFLiteConverter.from_keras_model(keras_model)
    tflite_model = converter.convert()
    open(save_mdl_path, "wb").write(tflite_model)

### 3. (optional) Any other info / logs

The CRNN TFlite model  (have rashape) can work on Android CPU, but set GPU will get erro: "java.lang.IllegalArgumentException: Internal error: Failed to apply delegate: Attempting to use a delegate that only supports static-sized tensors with a graph that has dynamic-sized tensors (tensor#40 is a dynamic-sized tensor)."

The tflite model can work on android GPU when  delete the layers.rshape and GRU .

I need reshspe layer for 3D to 2D dim. how I can do this?

thanks advance

**Comment by mohantym** [comment]
Hi @Yunlei-AI ! Could you please provide sample stand alone code to reproduce this issue?

**Comment by mohantym** [comment]
Hi @sachinprasadhs! Could you please take a look at this issue?

**Comment by sachinprasadhs** [comment]
GPU delegate doesn't support dynamic shapes.
You need to specify the static size to use the GPU delegate or you can use the dynamic shape with CPU.
Please look into the responses in the issue [here](https://github.com/tensorflow/tensorflow/issues/38036) for more insights on this. Thanks!
